README.md

```diff
@@ -39,7 +39,7 @@ You're welcome to add a new project in **verto.sh**, just follow these steps:
 
 ## How Does It Work?
 
-**verto.sh** uses Next.js, React and Typescript. The data shown on the website is loaded from the [data.json](data/data.json) file, which is generated by [data/get.ts](data/get.ts) by querying the GitHub API to fetch issues from the repositories listed in [config.json](config.json). The labels defined in [config.json](config.json) are used to filter issues for the repositories.
+**verto.sh** uses Next.js, React and Typescript. The data shown on the website is loaded from the [data.json](data/data.json) file, which is generated by [data/utils.ts](data/utils.ts) by querying the GitHub API to fetch issues from the repositories listed in [config.json](config.json). The labels defined in [config.json](config.json) are used to filter issues for the repositories.
```

Comment:
`utils.ts` contains the function that does that, but it might be a good time to explain that the data-fetching script, which can handle GitHub and GitLab, later writes that file.

Reply:
@lucavallin, I've made changes based on what I thought you'd like to have written. Please let me know if anything should change. Thank you!